tests/api/test_projects.py

```diff
@@ -348,3 +359,15 @@ def test_invalid_id(self, session, flask_api_url):
 
     def test_success(self, session, flask_api_url):
         assert True
+
+    def test_unauthorized_update(self, flask_api_url):
+        """ Tests for 403 when attempting to update a different users project
+        """
+        data = {"email": "user@knex.com", "password": "user"}
+        session = requests.Session()
+        response = session.post(flask_api_url + '/api/users/login', data=data)
+        assert response.status_code == 200
+        # this should be the id of a project from a different user
+        projectid = str(uuid.uuid4())
+        response = session.delete(flask_api_url + '/api/projects/' + projectid)
+        # assert response.status_code == 403
```

Comment:
I couldn't figure out how to search for any project where user is not the author quickly (and didn't want to POST one first to use the id) so @JamesComey please edit this later